src/assert_files.sh

```diff
@@ -51,3 +51,19 @@ function assert_is_file_empty() {
 
   state::add_assertions_passed
 }
+
+function assert_files_equals() {
+  local expected="$1"
+  local actual="$2"
+
+  if ! cmp -s "$expected" "$actual"; then
+    local label
+    label="$(helper::normalize_test_function_name "${FUNCNAME[1]}")"
+    state::add_assertions_failed
+    console_results::print_failed_test "${label}" "${expected}" "compared" "${actual}" \
+        "Diff" "$(diff -u "$expected" "$actual" || true)"
```

Comment:
thanks for working on my request <3.

does the diff command already return a exit code which can be used at the IF statement?
I think the results would be more consistent, if we don't use 2 different command to compare the files and to print the diff

Reply:
Very good point! I will change it 🧠